README.md

```diff
@@ -76,7 +76,8 @@ The following games are known to integrate GameMode support (meaning they don't
 Other apps which can integrate with GameMode include:
 * [ATLauncher](https://atlauncher.com/downloads) Minecraft launcher
 * [Cemu](https://cemu.info/) Wii U emulator.
-* GNOME Shell ([via extension](https://github.com/gicmo/gamemode-extension)) - indicates when GameMode is active in the top panel.
+* GNOME Shell ([via extension](https://github.com/gicmo/gamemode-extension)) - indicates when GameMode is active in the top panel. (not maintained)
```

Comment:
I'd rather remove this link entirely if it's an unmaintained project. I'll happily re-add it (and other alternative extensions) if @gicmo updates it.

Reply:
I didn't want to disrespect the original extension however it is not been updated since GNOME 42.